### PR TITLE
Feature | Issue #34 | React in TypeScript

### PR DESCRIPTION
**Pull request by hamza-akram**
## Description
This PR closes **#34**

Changes proposed in this pull request:
- Added Blogs/Articles and Talks for React in TypeScript.

## Checklist
- [x] Make sure you are making a pull request against the **main branch**
- [x] Check the PR title and branch naming structure matched with our [guidelines](../CONTRIBUTING.md) 
- Please check the type of change your PR introduces:
  - [x] New feature
  - [ ] Bug in content and documentation
  - [ ] Fix in content and documentation
  - [ ] Other changes

## Screenshots
Screenshot 2022-09-07 at 4 46 32 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,16 +30,21 @@
 </div>
 
 <!-- ABOUT THE PROJECT -->
+
```
Comment (nabeel-shakeel) [inline README.md:33]: @hamza-akram: Please remove the extra new lines added in the whole file

**Review by nabeel-shakeel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hamza-akram) [reply README.md:33]: Done!

Comment (nabeel-shakeel) [reply README.md:33]: @hamza-akram: Still this new line remains

Comment (hamza-akram) [reply README.md]: Done

**Review by nabeel-shakeel — APPROVED** [review]
Good work! Ready to ship 🚢

**Comment by nabeel-shakeel** [comment]
@hamza-akram: Thanks for your contribution ❤️

**Pull request merged**